backend/aquarium.py

```diff
@@ -70,6 +71,29 @@ def add_aquarium():
     return "Success", 200
 
 
+@app.route("/import-aquarium", methods=["POST"])
```

Comment:
This file is over 100 lines, You should add this into new python file. It's not cardinal error but best practices of programming to make files smaller than 100 lines (Uncle Bob said that)